bin/build_netkan_container.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+cp -v Dockerfile.netkan _build/repack/$BUILD_CONFIGURATION/.
```

Comment:
What do you think about doing these operations in a `_build/docker/inflator` folder instead of overloading `repack`? That way the docker stuff would be more isolated and not make implicit assumptions about how the rest of the build process works (i.e., that the `repack` folder will contain `netkan.exe`). And it seems more in the spirit of how the `_build` folder is currently organized, with each step in its own path.

Reply:
I think that's a wonderful idea! I'm not super across cake, but I know docker + bash!